docs/content/en/docs/topics/provision-a-cluster/_index.md

```diff
@@ -53,42 +53,35 @@ helm get manifest demo | grep size
 .
 .
 .
-  size: 1
-      initial_heap_size: "800M"
-      max_heap_size: "800M"
-    :[{\"expr\":\"sum(mcac_table_memtable_off_heap_size{cluster=~\\\"$cluster\\\"\
-    :\"A\"},{\"expr\":\"sum(mcac_table_memtable_on_heap_size{cluster=~\\\"$cluster\\\
-    ,\"description\":\"Total sizes of the data on distinct nodes\",\"fill\":0,\"gridPos\"\
-    datasource\":\"$PROMETHEUS_DS\",\"description\":\"Maximum JVM Heap Memory size\
-    \ (worst node) and minimum available heap size\",\"fill\":1,\"gridPos\":{},\"\
+    size: 1
+      initial_heap_size: 1G
+      max_heap_size: 1G
+      heap_size_young_generation: 1G
 ```
 
-Notice the value of `size: 1` from cassdc.yaml. This is the Cassandra DataCenter definition. 
+The value of `size: 1` is from cassdc.yaml, which is the Cassandra DataCenter definition. 
 
-To scale up, you could change the `size` to 3. Example with helm:
+To scale up, you could change the `size` to 3. In the following example, we'll also set the name `dc1`:
 
 ```bash
-helm upgrade demo k8ssandra/k8ssandra --set k8ssandra.size=3 --reuse-values
+helm upgrade k8ssandra k8ssandra/k8ssandra --set cassandra.datacenters\[0\].size=3,cassandra.datacenters\[0\].name=dc1
```

Comment:
It is easy enough to get bit by the merge semantics with arrays and maps that I think it would better to use an overrides file here. It is better in practice.

Reply:
I'd like to do this on the next round - for now, I'd like to get these corrections published in this PR. Thanks!@